robot_names:
  - X1
  - X2
  - X3
  - X4

button_robot_map:
  A: X1
  B: X2
  X: X3
  Y: X4

flashlight_service_suffixes:
  # X1 and X2
  - /right_headlight/enable
  - /center_right_headlight/enable
  - /center_left_headlight/enable
  - /left_headlight/enable
  # X3 and X4
  - /downward_flashlight/enable
  - /left_flashlight/enable
  - /right_flashlight/enable

robot_address_map:
  X1: X1
  X2: X2
  X3: X3
  X4: X4

select_led_service_suffixes:
  # X1
  - /left_lateral_front_led/enable
  - /left_lateral_rear_led/enable
  - /rear_left_led/enable
  - /right_lateral_front_led/enable
  - /right_lateral_rear_led/enable
  - /rear_right_led/enable
  # X2
  - /left_lateral_led/enable
  - /right_lateral_led/enable
  # X3 and X4
  - /left_led/enable
  - /right_led/enable

comm_led_service_suffixes:
  # X1 and X2
  - /rear_center_led/enable
  # X3 and X4
  - /center_led/enable
